superset/migrations/versions/80aa3f04bc82_add_parent_ids_in_dashboard_layout.py

```diff
@@ -38,8 +38,8 @@
 from superset import db
 
 # revision identifiers, used by Alembic.
-revision = '80669c0097a6'
-down_revision = 'c82ee8a39623'
+revision = '80aa3f04bc82'
```

Comment:
I think you should keep the revision ID unchanged.

Reply:
after rebase, I created script from scratch, so the system generated new file name.